ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/validation/LdesShaclValidator.java

```diff
@@ -33,16 +31,21 @@ public boolean supports(Class<?> clazz) {
 	@Override
 	public void validate(Object target, Errors errors) {
 		Member member = (Member) target;
-		validateShape(member, errors);
+		validateShape(member);
 	}
 
-	protected void validateShape(Member member, Errors errors) {
+	protected void validateShape(Member member) {
+		if (!initialized) {
+			if (validationConfig.isEnabled() && validationConfig.getShape() != null) {
```

Comment:
can't you create a 'shapes' bean in a configuration class and just inject it in this validator?